core/src/cats/sql_create.cc

```diff
@@ -1045,7 +1045,7 @@ bool BareosDb::CreateBaseFileAttributesRecord(JobControlRecord* jcr,
   Mmsg(cmd, "INSERT INTO basefile%lld (Path, Name) VALUES ('%s','%s')",
        (uint64_t)jcr->JobId, esc_path, esc_name);
 
-  retval = INSERT_DB(jcr, cmd);
+  retval = INSERT_DB(jcr, cmd) == 1 ? true : false;
```

Comment:
`x > 0` already is a boolean expression, no need to convert it again.
```suggestion
  retval = INSERT_DB(jcr, cmd) == 1;
```